pkg/database/postgresql_test.go

```diff
@@ -51,7 +51,8 @@ func TestPostgresql_DSN(t *testing.T) {
 				return s
 			},
 			expected: func(s *settings.Settings) string {
-				return "host=/tmp user=my_custom_user dbname=postgres password=mysecretpassword"
+				return fmt.Sprintf("postgres://%s:%s@?%s&%s&sslmode=%s",
+					"my_custom_user", "mysecretpassword", "/tmp", s.Port, s.SSLMode)
```

Comment:
Lets not use fmt.Sprintf anymore in these tests, I would like to have them very specific and see the actual string which is expected to be created. I know the above tests are using it, but can you change it there as well?